toolchain/gcc/Config.version

```diff
@@ -7,13 +7,20 @@ config GCC_VERSION_8
 	default y if arc
 	bool
 
+config GCC_VERSION_9
+	default y if GCC_USE_VERSION_9
+	default y if arc
```

Comment:
Please do not make this default on arc

Reply:
I was asked to add this. So should it be removed?

Reply:
I've remove it.